src/settings-ui/Settings.UI/SettingsXAML/Controls/KeyVisual/KeyVisual.xaml.cs

```diff
@@ -95,9 +95,23 @@ private void Update()
                 return;
             }
 
-            if (Content is string)
+            if (Content is string key)
             {
-                _keyPresenter.Style = (Style)Application.Current.Resources["DefaultKeyCharPresenterStyle"];
+                switch (key)
+                {
+                    case "Copilot":
+                        _keyPresenter.Style = (Style)Application.Current.Resources["OfficeKeyCharPresenterStyle"];
+                        break;
+
+                    case "Office":
+                        _keyPresenter.Style = (Style)Application.Current.Resources["CopilotKeyCharPresenterStyle"];
```

Comment:
Flipped for style in black th office and couoot

Reply:
Doh.. good catch